worker/src/test/java/gov/cms/ab2d/worker/processor/JobPreProcessorIntegrationTest.java

```diff
@@ -61,7 +61,7 @@ void setUp() {
     @Test
     @DisplayName("When a job is in submitted status, it can be put into progress upon starting processing")
     void whenJobIsInSubmittedStatus_ThenJobShouldBePutInProgress() {
-        var processedJob = cut.preprocess("S001");
+        var processedJob = cut.preprocess("S00000");
```

Comment:
An extra `0` here and onward.

Reply:
changed it to have 4 zeroes instead of 5 zeroes